amazon-sqs/5.6/modules/ROOT/pages/amazon-sqs-connector-config-topics.adoc

```diff
@@ -0,0 +1,34 @@
+= Amazon SQS Connector 5.6 Additional Configuration Information - Mule 4
+:page-aliases: connectors::amazon/amazon-sqs-connector-config-topics.adoc
+
```

Comment:
I would add like an intro sentence to reference the upcoming configuration sections, something like:
_Set additional configurations for Anypoint Connector for Amazon SQS (Amazon SQS Connector), such as how to receive messages in cluster mode and create the FIFO queue_....